Styles/grid.css

```diff
@@ -0,0 +1,28 @@
+.grid-2{
+    display: grid;
+    background-color: lightgray;
+    grid-gap: 2rem;
+    justify-content: center;
+    grid-template-columns: repeat(2, 16rem);
+}
+.grid-3{
+    display: grid;
+    background-color: lightgray;
+    grid-gap: 2rem;
+    justify-content: center;
+    grid-template-columns: repeat(3, 16rem);
+}
```

Comment:
**display, grid-gap, justify-content, background** properties are repeated. You can assign these properties to one class and utilise it.